cmd/buildah/images.go

```diff
@@ -299,35 +304,64 @@ func outputImages(ctx context.Context, systemContext *types.SystemContext, store
 			imageID = shortID(image.ID)
 		}
 
+		repos := imageReposToMap(image.Names)
+		if len(repos) == 0 {
+			repos = map[string][]string{"": []string{""}}
+		}
 	outer:
-		for name, tags := range imageReposToMap(image.Names) {
-			for _, tag := range tags {
-				if !matchesReference(name+":"+tag, argName) {
+		for repo, tagsAndDigests := range repos {
+			var imageDigests []string
```

Comment:
This could be moved out of this loop as well.

Reply:
Ah, right.  Moving it.